examples/OpenTelemetry/DistributedTracing/Client/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Diagnostics;
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace Client
+{
+    internal class Program
+    {
+        private const string WeatherApiUri = "http://localhost:5284";
+
+        private const string WeatherForecastEndpointUri = WeatherApiUri + "/" + "WeatherForecast";
+
+        private static readonly ActivitySource ClientActivity = new ActivitySource("Weather Forecast Client Request");
+
+        static async Task Main(string[] args)
+        {
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .SetResourceBuilder(ResourceBuilder.CreateDefault().AddService(
+                    serviceName: "Weather Forecast Client",
+                    serviceVersion: "1.0.0"))
+                .AddSource(ClientActivity.Name)
+                //.AddOtlpExporter(opt => opt.Endpoint = new Uri("http://localhost:4317")) // uncomment if you want to use an OTPL exporter like Jaeger
+                .Build();
+            var cts = new CancellationTokenSource();
+            var task = Task.Run(async () =>
+            {
+                await Task.Delay(1000, cts.Token).ConfigureAwait(false);
+                using var httpClient = new HttpClient();
+                while (!cts.IsCancellationRequested)
+                {
+                    try
+                    {
+                        using (var _ = ClientActivity.StartActivity(ActivityKind.Client))
+                        {
+                            await Console.Out.WriteLineAsync("TraceId: " + Activity.Current?.TraceId + Environment.NewLine + "Sending request.").ConfigureAwait(false);
+                            var forecastResponse = await httpClient.GetAsync(WeatherForecastEndpointUri, cts.Token).ConfigureAwait(false);
+
+                            if (forecastResponse.IsSuccessStatusCode)
+                            {
+                                var content = await forecastResponse.Content.ReadAsStringAsync(cts.Token).ConfigureAwait(false);
+                                //var forecast = JsonSerializer.DeserializeAsync<WeatherForecast>(content).ConfigureAwait(false);
```

Comment:
I think this comment is supposed to be deleted, no?

Reply:
nice catch, fixed